build/AzurePipelinesTemplates/MUX-WACKTests-Job.yml

```diff
@@ -68,6 +66,21 @@ jobs:
       contents: '*.xml'
       targetFolder: $(resultLogStagingPath)
 
+  - task: PkgESWACKTests@12
+    displayName: Run WACK tests for UwpAppWithFullTrustProcess
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    condition: succeededOrFailed()
+    inputs:
+      PackageFilePath: '$(appxPackagesPath)\UwpAppWithFullTrustProcess_package_Test\UwpAppWithFullTrustProcess_package.msix'
+  - task: CopyFiles@2
+    displayName: 'Copy results for AppThatUsesMUXIndirectly'
```

Comment:
Typo: should be UwpAppWithFullTrustProcess

Reply:
Good catch.